ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleTagManagerBridge.m

```diff
@@ -42,11 +42,23 @@ @implementation RCTGoogleTagManagerBridge
 
     [TAGContainerOpener openContainerWithId:containerId
                                  tagManager:self.tagManager
-                                 openType:kTAGOpenTypePreferFresh
+                                 openType:kTAGOpenTypePreferNonDefault
```

Comment:
This changes behaviour. [I see from docs that](https://developers.google.com/tag-manager/ios/v3/) `kTAGOpenTypePreferNonDefault` is preferred, but is there a reason why this was changed? Also the Java-code still uses `loadContainerPreferFresh`?

Reply:
I had a problem refreshing the container, it always returned the stale container with `PreferFresh`, quite the opposite of what the documentation mentioned (Google does not care much about legacy reference documentations; Broken links, misleading samples, etc.). So, I changed this line and it "seemd" to work, but turns out it didn't have any effect. It is the change of the flag that somehow cause refresh, not the actual value. After days of testing, I realized that we cannot rely on the freshness of container at all and we need to bundle the container with each update. Even with the `refreshContianer` method it always complained about too many request being sent and ignored the command, not matter how many minutes and hours we put between the calls. Anyhow, I'll be reverting this back to its previous valu